man/kernel-install.xml

```diff
@@ -164,6 +164,9 @@
 
     <para>If <varname>MACHINE_ID=</varname> is set and nonempty, it will be used as <replaceable>MACHINE-ID</replaceable>,
     overriding any automatic detection attempts.</para>
+
+    <para><varname>KERNEL_INSTALL_MACHINE_ID=</varname> is set for the plugins to the desired <replaceable>MACHINE-ID</replaceable>.</para>
```

Comment:
maybe mention that this one might be either a 32char hex string or the special word "Default"

Reply:
Done